File: 2025-12-December-GFG-POTD/December 19 - Bus Conductor/Solution.py
class Solution:
    def findMoves(self, chairs, passengers):
        # Sort both arrays
        chairs.sort()
        passengers.sort()
        
        moves = 0
        
        # Sum absolute differences
        for i in range(len(chairs)):
            moves += abs(chairs[i] - passengers[i])
        
        return moves

File: 2025-12-December-GFG-POTD/December 19 - Bus Conductor/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    int findMoves(vector<int>& chairs, vector<int>& passengers) {
        // Sort both arrays
        sort(chairs.begin(), chairs.end());
        sort(passengers.begin(), passengers.end());
        
        int moves = 0;
        
        // Sum absolute differences
        for (int i = 0; i < chairs.size(); i++) {
            moves += abs(chairs[i] - passengers[i]);
        }
        
        return moves;
    }
};
